frontend/utility/MissingFilesPathItemDelegate.cpp

```diff
@@ -68,15 +63,12 @@ QWidget *MissingFilesPathItemDelegate::createEditor(QWidget *parent, const QStyl
 
 	container->connect(browseButton, &QToolButton::clicked, browseCallback);
 
-	// The "clear" button is not shown in input cells
-	if (isOutput) {
-		QToolButton *clearButton = new QToolButton();
-		clearButton->setText("X");
-		clearButton->setSizePolicy(buttonSizePolicy);
-		layout->addWidget(clearButton);
+	QToolButton *clearButton = new QToolButton();
```

Comment:
Don't we have an actual glyph that we can use for this vs. "faking" it with an capital "x" (which will always look a bit out of place as it will use a non-monospace font) or _if_ text rendering is used, wouldn't it be preferable to use a more uniform Unicode glyph like "✕" or "✖"?